README.rst

```diff
@@ -0,0 +1,126 @@
+using Binder!
+For questions on PyMC3, head on over to our `PyMC Discourse <https://discourse.pymc.io/>`__ forum.
+
+Getting started
+===============
+
+If you already know about Bayesian statistics:
+----------------------------------------------
+
+
+-  `API quickstart guide <http://docs.pymc.io/notebooks/api_quickstart>`__
+-  The `PyMC3 tutorial <http://docs.pymc.io/notebooks/getting_started>`__
+-  `PyMC3 examples <https://docs.pymc.io/nb_examples/index.html>`__ and the `API reference <http://docs.pymc.io/api>`__
+
+
+
+Learn Bayesian statistics with a book together with PyMC3:
+----------------------------------------------------------
+
+-  `Probabilistic Programming and Bayesian Methods for Hackers <https://github.com/CamDavidsonPilon/Probabilistic-Programming-and-Bayesian-Methods-for-Hackers>`__: Fantastic book with many applied code examples.
+-  `PyMC3 port of the book "Doing Bayesian Data Analysis" by John Kruschke <https://github.com/aloctavodia/Doing_bayesian_data_analysis>`__ as well as the `second edition <https://github.com/JWarmenhoven/DBDA-python>`__: Principled introduction to Bayesian data analysis.
+-  `PyMC3 port of the book "Statistical Rethinking A Bayesian Course with Examples in R and Stan" by Richard McElreath <https://github.com/pymc-devs/resources/tree/master/Rethinking>`__
+-  `PyMC3 port of the book "Bayesian Cognitive Modeling" by Michael Lee and EJ Wagenmakers <https://github.com/pymc-devs/resources/tree/master/BCM>`__: Focused on using Bayesian statistics in cognitive modeling.
+-  `Bayesian Analysis with Python  <https://www.packtpub.com/big-data-and-business-intelligence/bayesian-analysis-python-second-edition>`__ (second edition) by Osvaldo Martin: Great introductory book. (`code <https://github.com/aloctavodia/BAP>`__ and errata).
+
+PyMC3 talks
+-----------
+
+There are also several talks on PyMC3 which are gathered in this `YouTube playlist <https://www.youtube.com/playlist?list=PL1Ma_1DBbE82OVW8Fz_6Ts1oOeyOAiovy>`__
+and as part of `PyMCon 2020 <https://discourse.pymc.io/c/pymcon/2020talks/15>`__
+
+Installation
+------------
+
+To install PyMC3 on your system, see its `installation section here <https://github.com/pymc-devs/pymc3#installation>`__
+
+Citing PyMC3
+============
+
+Salvatier J., Wiecki T.V., Fonnesbeck C. (2016) Probabilistic programming
+in Python using PyMC3. PeerJ Computer Science 2:e55
+`DOI: 10.7717/peerj-cs.55 <https://doi.org/10.7717/peerj-cs.55>`__.
+
+Papers citing PyMC3
+-------------------
+
+See `Google Scholar <https://scholar.google.de/scholar?oi=bibs&hl=en&authuser=1&cites=6936955228135731011>`__ for a continuously updated list.
+
+Contributors
+============
+
+See the `GitHub contributor
+page <https://github.com/pymc-devs/pymc3/graphs/contributors>`__. Also read our `Code of Conduct <https://github.com/pymc-devs/pymc3/blob/master/CODE_OF_CONDUCT.md>`__ guidelines for a better contributing experience.
```

Comment:
The link should be to the contributor page for pymc-examples, code of conduct is fine like this for now